include/pmacc/particles/memory/buffers/StackExchangeBuffer.hpp

```diff
@@ -150,12 +161,24 @@ namespace pmacc
 
         size_t getHostParticlesCurrentSize()
         {
-            return stack.getHostBuffer().getCurrentSize();
+            size_t result = 0u;
```

Comment:
Why do you introduce the in-between variable and not use `return` in both cases?